numpy/core/include/numpy/ndarraytypes.h

```diff
@@ -354,20 +354,9 @@ struct NpyAuxData_tag {
 #define NPY_USE_PYMEM 1
 
 #if NPY_USE_PYMEM == 1
-   /* numpy sometimes calls PyArray_malloc() with the GIL released. On Python
-      3.3 and older, it was safe to call PyMem_Malloc() with the GIL released.
-      On Python 3.4 and newer, it's better to use PyMem_RawMalloc() to be able
-      to use tracemalloc. On Python 3.6, calling PyMem_Malloc() with the GIL
-      released is now a fatal error in debug mode. */
```

Comment:
Maybe I should adapt and leave part of this message?

/* numpy sometimes calls PyArray_malloc() with the GIL released. 
   On Python 3.4 and newer, it's better to use PyMem_RawMalloc() to be able
   to use tracemalloc. After Python 3.6, calling PyMem_Malloc() with the GIL
   released is now a fatal error in debug mode. */

Reply:
I think leaving behind a comment like `/* use the Raw versions which are safe to call with the GIL released */`

Reply:
That's a concise comment. added (along with your other comment suggestion)